tasks/src/text-to-video/data.ts

```diff
@@ -36,7 +36,24 @@ const taskData: TaskDataCustom = {
 		outputs: [],
 	},
 	isPlaceholder: true,
-	metrics:       [],
+	metrics:       [
+		{
+			description: "Inception Score uses separate image classification model (Inception-V3, C3D) predict class labels and evaluates how distinct and diverse they are; higher scores are better.",
+			id: "is",
+		},
+		{
+			description: "Frechet Inception Distance uses a separate model to embed images and compares the distribution of generated image representations with the distribution of real image representations; smaller scores are better.",
```

Comment:
```suggestion
			description: "Frechet Inception Distance uses an image classification model to embed images and compares the distribution of generated image representations with the distribution of real image representations. The smaller the score is better the video generation.",
```
feel free to swap image classification if it's not the case. also, is it the model that compares the distribution? can you clarify? also how does it embed?

Reply:
Added some more details on how it embeds and compares the distributions :)